homeassistant/data_entry_flow.py

```diff
@@ -188,7 +193,7 @@ def _map_error_to_schema_errors(
     schema_errors[path_part_str] = error.error_message
 
 
-class BaseFlowManager(abc.ABC, Generic[_FlowResultT]):
+class BaseFlowManager(abc.ABC, Generic[_FlowResultT, _HandlerT]):
```

Comment:
Is it possible to access the generic parameter of the `_FlowResultT` to avoid adding a second generic parameter here?

Reply:
As discussed on Discord, that would need something like https://github.com/python/typing/issues/548